### ISSUE TITLE
Establishing connection to server

### ISSUE THREAD

**Issue by aaronmee** [jellyfin/jellyfin-android#1393]
### Describe the bug

Hi,
I've been running a jellyfin instance on my diy server for 3 months without an issue. 
Now I installed a new instance on my `raspberry pi 5` as a `linuxserver.io docker container`.
Accessing the instance via browser works without an issue. Accessing it with the android app though doesn't work. I just get an `connection couldn't established` error. 

Below are the server logs of me trying to connect to the server with the android app. The handshake log appears after I closed the app.
```
[2024-05-21 20:45:06.215 +02:00] [INF] [69] Emby.Server.Implementations.HttpServer.WebSocketManager: WS "192.168.0.230" request
[2024-05-21 20:45:15.995 +02:00] [WRN] [69] Emby.Server.Implementations.HttpServer.WebSocketConnection: WS "192.168.0.230" error receiving data: "The remote party closed the WebSocket connection without completing the close handshake."
[2024-05-21 20:45:16.004 +02:00] [INF] [69] Emby.Server.Implementations.HttpServer.WebSocketManager: WS "192.168.0.230" closed
```

### Application version

2.6.1

### Where did you install the app from?

Google Play

### Device information

Samsung Galaxy S24

### Android version

Android 14 (OneUI 6.1)

### Jellyfin server version

10.9.2

### Which video player implementations does this bug apply to?

- [ ] Web player (default)
- [ ] Integrated player (ExoPlayer)
- [ ] External player (VLC, mpv, MX Player)

**Comment by itszeekay** [comment]
Experiencing this issue now too. Both on local IP:Port http, and via https domain tunnel. Two different networks entirely, web and windows app work fine.

**Comment by nielsvanvelzen** [comment]
osing as duplicate of https://github.com/jellyfin/jellyfin-android/issues/1382. This is caused by an issue in jellyfin-web that is fixed in 10.9.3.

**Comment by aaronmee** [comment]
Alright, so imma wait until 10.9.3 releases...
